.gitignore

```diff
@@ -54,3 +54,5 @@ etc/
 var/
 # Go compiled tests
 *.test
+# OSX trash
+.DS_Store
```

Comment:
I'm not sure what is causing these files to be created, so not sure if the other extraneous MacOS files should also be included.

Reply:
According to this [site](https://buildthis.com/ds_store-files-and-why-you-should-know-about-them/), we should avoid adding .DS_Store files into repo. In other well-known git repositories, such as kubernetes and helm, .DS_Store files are added to the `.gitignore` file.

Reply:
If we want to follow Kubernetes, we should add the other MacOS (I would update the comment to the rename MacOS) entry as well:
```
# MacOS leaves these everywhere on SMB shares
._*

# MacOS Desktop Services Store
.DS_Store
```

Reply:
LGTM. Thanks.